Makefile

```diff
@@ -316,10 +316,10 @@ endif
 
 # Debugging
 ifeq ($(DEBUG), 1)
-	COMMON_FLAGS += -DDEBUG -g -O0
+	COMMON_FLAGS += -DDEBUG -g -O0 -std=c++11
 	NVCCFLAGS += -G
 else
-	COMMON_FLAGS += -DNDEBUG -O2
+	COMMON_FLAGS += -DNDEBUG -O2 -std=c++11
```

Comment:
@mbassov I finished training the base net w/ this C++ change.  We get an accuracy of `0.4903` which I think is close to the accuracy number you got and also close to the base net accuracy I observed earlier.